test/testutils/syncenginetestutils.cpp

```diff
@@ -1212,8 +1212,10 @@ FakeReply::FakeReply(QObject *parent)
     : QNetworkReply(parent)
 {
     setRawHeader(QByteArrayLiteral("Date"), OCC::Utility::formatRFC1123Date(QDateTime::currentDateTimeUtc()).toUtf8());
+#if QT_VERSION >= QT_VERSION_CHECK(6, 3, 0)
     // emulate the real world
```

Comment:
Hm, this appears to be a useless comment when you are not involved ;-)

Reply:
Its our testing moc code :D